Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM ubuntu:16.04
```

Comment:
I don't know which Keen image would be appropriate.  That would likely be wiser.  Do you have a suggestion?